components/utilities/breadCrumbs.module.css

```diff
@@ -14,6 +14,10 @@
 }
 
 .Separator {
-  @apply dark:text-gray-70 ml-2;
+  @apply ml-2;
   @apply text-gray-50 !important;
 }
+
+:global(.dark) .Separator {
+  @apply text-gray-70 !important;
+}
```

Comment:
Is this part of this PR on purpose?

Reply:
It is! When we refactored the `breadCrumb` component, I didn't realize the `/` separator changed its color slightly on dark mode, so I snuck it in here 😄